elastic/kibana
# [Uptime] [Synthetics Integration] fix tls settings editing

## Summary

Fixes #109280 

Previously, TLS settings that were saved with the integration policy were lost when editing the integration policy. This PR ensures that TLS settings can be edited. 

### Testing
1. Create a synthetics integration with TLS enabled and some TLS settings filled in. This should ideally be done for both http and tcp monitors.
2. Edit the policy
3. Observe your TLS settings are in place
